integration_test/startapp_test.dart

```diff
@@ -0,0 +1,8 @@
+import 'src/common_methods.dart';
+import 'package:patrol/patrol.dart';
+
+void main() {
+  patrolTest(nativeAutomation: true, 'Starting application', ($) async {
```

Comment:
We have a convention that test description is always first argument, and test callback is last, so put `nativeAutomation` in the middle

Reply:
I've corrected the arguments order so that test description would be first